app/models/card.rb

```diff
@@ -29,6 +31,21 @@ def check_translation(user_input)
 
   protected
 
+  def text_comparison user_input, original_text
```

Comment:
Assignment Branch Condition size for text_comparison is too high. [15.68/15]<br>Use def with parentheses when there are parameters.